### PR TITLE
tr: day 1 morning translation

### PR DESCRIPTION
**Pull request by Enes1313**
Translated day-1-morning
Part of #500

### CODE REVIEW

**Comment by mgeisler** [comment]
Thanks @Enes1313! Let me try and find someone who can help review this!

**Comment by Enes1313** [comment]
If you don't mind, I told a few friends to "come and review". I don't know when they will arrive :) I think you will find someone until one of them comes :)

File: po/tr.po [hunk 1]
```diff
@@ -3220,193 +3228,235 @@ msgid ""
...
 
 #: src/control-flow-basics/if.md
 msgid "\"biggish\""
-msgstr ""
+msgstr "\"büyükçe\""
 
 #: src/control-flow-basics/if.md
 msgid "\"huge\""
-msgstr ""
+msgstr "\"büyük\""
```
Comment (alerque) [inline po/tr.po:3345]: This probably shouldn't be the same as the translation for 'large'. Consider 'kocaman'.

File: po/tr.po [hunk 2]
```diff
@@ -3882,18 +3955,18 @@ msgid ""
 "covered later."
 msgstr ""
```
Comment (alerque) [inline po/tr.po:3956]: A number of strings are not translated here. I don't know if 100% coverage is required or what, but the PR says "fully" and there are a good chunk of empty strings near here.

**Review by alerque — COMMENTED** [review]
I did not review this thoroughly, just skimmed through it and did some smoke checks. Most of what I landed on seemed pretty good, but there are quite a few empty strings.

File: po/tr.po (hunk 1 above)
Comment (Enes1313) [reply po/tr.po:3345]: Thank you. Yes, it shouldn't be the same as "large" but I couldn't find a nice Turkish word instead. The word "kocaman" doesn't sound good. Maybe I could choose words like "dev" or "devasa". I need to consult a few people about this. Let me make a change based on the answers I get. I will make changes as soon as possible according to the answers I receive.

File: po/tr.po (hunk 2 above)
Comment (alerque) [reply po/tr.po:3956]: Okay got it. I wasn't following where all the pieces were coming from or what the expectations on completeness were, just spot checking.

Kolay gelsin.

Comment (Enes1313) [reply po/tr.po:3956]: Teşekkür ederim :)

Comment (mgeisler) [reply po/tr.po:3956]: Empty strings are okay in a PO file — that just means that the entry isn't translated yet. The English text will be used in the interim.

**Comment by Enes1313** [comment]
@alerque can you review again, please

**Comment by alerque** [comment]
From a quick once over it looks good.

By the way in the future it would be a *LOT* easier to review PRs if you did multiple commits and first added a commit that updated the existing PO file with the current message ids and updated source, then after that have a commit that only adds/changes the actual translations. This PR is pretty hard to weed through what is changes that came in based on PO source updates and what was actually done as translation work.

**Comment by Enes1313** [comment]
Oh, I have understood. Thank you.

**Comment by alerque** [comment]
I'm not a maintainer and the review approval mechanism is disabled on this project for non-maintainers otherwise I would use it.

**Review by mgeisler — APPROVED** [review]
Thanks @Enes1313 for the work here and thanks @alerque for reviewing it!

**Comment by alerque** [comment]
Sure.

**Comment by mgeisler** [comment]
> Sure.

Thanks! You've now been invited and should be able to approve future PRs! You're welcome to add yourself to the `.github/CODEOWNERS` file so that you're automatically assigned to the Turkish PRs.

**Pull request merged**